app/templates/courses/teacher-classes-view.ozar.pug

```diff
@@ -73,24 +74,25 @@ block content
         a.pull-right(href='/outcomes-report/teacher/' + me.get('_id') target="_blank" data-i18n='outcomes.view_outcomes_report')
 
     .classes.container
-      // Loop each class
+      // Loop each owned class
       each classroom in view.classrooms.models
         unless classroom.get('archived')
-          if classroom.isOwner()
+          if classroom.isOwner() && !(me.isAdmin() && classroom.get('ownerID') != view.teacherID)
```

Comment:
why isAdmin condition? When does that come into picture?

Reply:
Admins can look at CoCo classes without becoming the teacher. If they do that, the my-classrooms vs. shared-classrooms distinction should still work properly. They weren't differentiating, and archived classes were getting mixed up in it.

The .ozar version of this file isn't used, but it'll be merged away soon.